game/script-jokes.rpy

```diff
@@ -735,3 +755,61 @@ label joke_cheesy_pizza:
     extend 1nsqpo " {i}that cheesy{/i}."
     
     return
+
+label joke_veggie_mood:
+    n 1nslsssbl "Heh.{w=0.75}"
+    extend 1fcssssbl " Let's see who {i}this{/i} reminds you of, [player]."
+    n 1ccsajsbl "A-{w=0.2}hem!"
+    n 1tnmbo "Do you have {i}any{/i} idea when vegetarians get mood swings?{w=0.75}"
+    n 1nllss "Well...{w=0.5}{nw}"
+    extend 1fcssmesm " I do."
+    n 1fcsbg "...When they're out of their {i}gourd{/i},{w=0.5}{nw}"
+    extend 1fchbg " obviously!"
+    
+    return
+
+label joke_scarecrow_award:
+    n 1fsqfl "...Why did the scarecrow get an award?"
+    n 1ncsemesi "..."
+    n 1fcsbo "Because...{w=1}"
+    n 1fsqfl " it was {i}outstanding{/i}{w=0.75}{nw}" 
+    extend 1nsrca " in its field."
+    
+    return
+
+label joke_sundae_school:
+    n 1ullaj "You know...{w=0.75}{nw}"
+    n 1tnmbo "I've been thinking about school a bunch lately."
+    n 1ulraj "I mean,{w=0.5}{nw}"
+    n 1unmfl " there were {i}so{/i} many schools I could've gone to -{w=0.5}{nw}"
+    extend 1fspbg " one even did culinary classes!"
+    n 1fcsan "But the schedule meant going in on a {i}weekend{/i}!{w=0.5}{nw}"
+    extend 1fllwr " Who {i}does{/i} that?!"
+    n 1fcsem "Ugh..."
+    n 1flrfl "Talk about a...{w=1}{nw}"
+    extend 1fsqss " {i}sundae{/i}{w=0.75}{nw}" 
+    extend 1uchgn " school."
+    
+    return
+
+label joke_burned_tongue:
+    n 1ullaj "You know,{w=0.75}{nw}"
+    extend 1tnmbo " I never understood why hipsters hung around coffee shops so much."
+    n 1nsrbo "Like...{w=0.3} what's so fun about sitting around sipping from a cup all day?"
+    n 1nllfl "Only yesterday I saw one of them in a café lately.{w=0.3}{nw}"
+    extend 1unmaj " They burned their tongue.{w=0.3}{nw}"
+    extend 1tnmss " Wondering why?{w=0.3}"
```

Comment:
```suggestion
    n 1nllfl "In fact,{w=0.2} only yesterday I saw one of them in a café..."
    n 1unmaj "And they kept burning their tongue!{w=0.5} Like it was on purpose or something!{w=0.75}{nw}"
    extend 1tnmss " Wondering why,{w=0.2} [player]?"
```

"Only yesterday I saw one of them in a café lately" comes off a bit strange here - it's like Nat is saying yesterday she saw one in the past! I'd suggest this tweak so it makes a little more sense.

Having Nat announce that they burned their tongue in this way feels a bit sudden: having a bit of lead up to it makes it feel more like a story Natsuki is telling to lead up to the joke! Nat also generally targets her questions towards the player, so I would adjust her line where she prompts the player to reflect this.

Lastly, .3 is a bit short here for pauses: when extending, it'd be best to stick to .5 for a comma/dash, .75 for a full stop/exclamation point/question mark, and 1 for an ellipsis so it's all consistent with other writing, and to give the player enough time to register the expression changes: I'll get this on the writing guide.